Cargo.toml

```diff
@@ -28,6 +28,7 @@ name = "packs"
 path = "src/lib.rs"
 
 [dependencies]
+anyhow = { version = "1.0.75", features = [] } # for error handling
```

Comment:
If we're OK with bringing `anyhow` in, I'll follow this PR up with one that replaces the existing `std::result` usages